src/timeMachine/components/RawFluxDataGrid.tsx

```diff
@@ -119,6 +119,9 @@ export default class extends PureComponent<Props, State> {
 
   private getCellData(row, column) {
     const {data} = this.props
-    return data[row][column]
+    if (data[row] && data[row][column]) {
```

Comment:
not to be a hipster or anything, but `return this.props.data[row]?[column] || ''` maybe?

Reply:
how bout `private getCellData = (row, column) => (this.props.data[row] ? [column] : '')`

Reply:
actually I think this is the best we can do: `private getCellData = (row, column) => (this.props.data[row] ? this.props.data[row][column] : '')`